src/Transactions/TransactionDetails/TransactionInformation/SourceValue/useSourceValue.js

```diff
@@ -0,0 +1,31 @@
+import { useQuery } from 'react-query';
+
+import { useOkapiKy } from '@folio/stripes/core';
+import {
+  LINES_API,
+  INVOICES_API,
+} from '@folio/stripes-acq-components';
+
+import { TRANSACTION_SOURCE } from '../../../constants';
+
+export const useSourceValue = (transaction) => {
+  const ky = useOkapiKy();
+  const isInvoiceSource = transaction.source === TRANSACTION_SOURCE.invoice;
+
+  const { isLoading: isInvoiceLoading, data: invoice } = useQuery(
+    transaction.id,
```

Comment:
I'm not sure that usage id as a key is the best. usually it's something complex, like name of the module, part of the module or entity and a param (id in this case).